models/fastText_expirements.ipynb

```diff
@@ -0,0 +1,156 @@
+{
```

Comment:
<p>can you add a df.head() here so we can see that the dataframe looks like?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/github-labeler/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>